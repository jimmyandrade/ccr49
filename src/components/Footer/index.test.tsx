import { create } from "react-test-renderer";
import {Footer} from "./index";

describe("<Footer /> component", () => {
  it("should match snapshot", () => {
    const testRenderer = create(<Footer title={"Test Title"} />);
    expect(testRenderer).toMatchSnapshot();
  });
});