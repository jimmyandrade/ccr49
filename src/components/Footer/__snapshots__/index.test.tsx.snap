// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component should match snapshot 1`] = `
<footer
  className="MuiBox-root MuiBox-root-2 Footer-root-1"
>
  <div
    className="MuiContainer-root MuiBox-root MuiBox-root-3 MuiContainer-maxWidthLg"
  >
    <p
      className="MuiTypography-root MuiTypography-caption MuiTypography-paragraph"
    >
      <time
        dateTime="2021"
      >
        2021
      </time>
       
      <strong>
        Test Title
      </strong>
       
       - Valorize seu caminho
      <br />
      Crescimento. Evolução. Autonomia.
    </p>
    <p
      className="MuiTypography-root MuiTypography-caption MuiTypography-paragraph"
    >
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
        href="#"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Política de privacidade
      </a>
       
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineAlways MuiTypography-colorInherit"
        href="#"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Proteção e uso de dados
      </a>
    </p>
  </div>
</footer>
`;
